plane/source/docs/aborting-autolanding.rst

```diff
@@ -0,0 +1,45 @@
+.. _aborting-autolanding:
+
+===========================
+How to Abort an Autolanding
+===========================
+A landing-abort mechanism is provided to allow you to abort a landing sequence in a safe, controlled, and expected way. Custom abort behaviour can be pre-programmed as part of the mission or you can use the default abort mechanism. To enable this feature set param LAND_ABORT_THR=1.
+ 
+There are three steps to this feature:
+
+1. :ref:`Trigger an abort <trigger_an_abort>`
+#. :ref:`The behavior during the abort <behavior_during_the_abort>`
+#. :ref:`The mission state after the abort completes <mission_state_after_an_aborted_landing_completes>`
+
+
+.. _trigger_an_abort:
+
+Step 1) Abort land triggers
+---------------------------
+The are three ways to trigger an auto-landing abort. All of them will only work while in AUTO mode and currently executing a ``LAND`` waypoint mission item:
```

Comment:
This implies that the landing abort will not work if the auto land is triggered by RTL or QRTL. I'm pretty sure it will, but I guess I could be wrong.

Reply:
QRTL does not trigger an autolanding....an rtl can IF rtl autoland is setup to execute a do-land-start sequence....the abort only applies to the nav_land waypoint part of a mission sequence whether standalone or part of a do land start sequence

Reply:
Ok - so my point is that it currently says it will only work in AUTO mode, but actually it will also work in RTL mode if certain conditions are met ...

Reply:
with rtl autloand it switches from rtl to auto ,running a mission starting at do land start and follows the mission...its not in rtl any more

Reply:
Hmm - well let me put it this way - I have a "RTL" mode switch on my transmitter. I have several times switched to RTL, watched as the plane attempted to do an auto land without realizing that it had switched to AUTO, until you told me that just now, I always thought the auto landing was happening in RTL, hence my comment.

So I'm thinking this needs to be addressed at line 41. "The auto landing _usually_ begins after reaching the last navigation waypoint." then add a new paragraph after line 41 
"The auto landing can also be triggered by RTL as explained on (link to do land start page), in which case the mode will switch to AUTO."